Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/Result.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.AmazonS3.DownloadObject
```

Comment:
Please use one-line namespace definition. Also, the namespace should be `Frends.AmazonS3.DownloadObject.Definitions.`.